docs/core-concepts/reusable-content-pieces/reusable-content-with-pieces.md

```diff
@@ -4,13 +4,19 @@ When you create content with the rich text editor, or images widget, that conten
 
 ## A directory of people: working with pieces
 
-Let's say you want to create a directory of people who work for a company. People are "global content": they are useful to display here and there all over the site, they aren't tied down to one page. The `apostrophe-pieces` module provides a great starting point to create many types of global content. You'll extend it to make your own `people` module. You can extend `apostrophe-pieces` many times in the same project.
+Let's say you want to create a directory of people who work for a company. People are "global content": they are useful to display in multiple places all over the site and they aren't tied down to one page. The `apostrophe-pieces` module provides a great starting point to create many types of global content. You'll extend it to make your own `people` module. You can extend `apostrophe-pieces` many times in the same project.
 
-::: tip NOTE
-**"What about users?"** Yes, you already have a "Users" menu on your admin bar. And yes, users are powered by pieces. But we've found that confusing website editors with the publicly visible staff directory tends to cause problems in the long run. Plus, this way, it's a teachable moment. :\)
+::: details What about "users"?
```

Comment:
This is a collapsed block from Vuepress. There are a lot of those green tip blocks already on the page. This seemed less critical, so a good match for this type instead.